R/print_hierplane.R

```diff
@@ -1,9 +1,4 @@
 #' @export
-print.hierplane_dataframe <- function(x, ...) {
-  cat("<hierplane_tree object: from hp_dataframe>", ...)
-}
-
-#' @export
-print.hierplane_spacyr <- function(x, ...) {
-  cat("<hierplane_tree object: from hp_spacyr>", ...)
+print.hierplane_tree <- function(x, ...) {
+  cat("<hierplane_tree object>", ...)
```

Comment:
Do we need to update the attribute assignment in hp_dataframe?

Reply:
Hm, I think we can probably just give everything hierplane_tree, no need for the specific ones. What do you think?

Reply:
I have remove the hierplane_spacyr and hierplane_dataframe attributes, so now everything is just hierplane_tree.

Reply:
Please mark this convo as unresolved if you disagree with this removal.